scripts/travis/pylint_harness.sh

```diff
@@ -15,11 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-disabled="--disable=similarities,invalid-name,too-many-statements,too-many-arguments,too-many-locals,too-few-public-methods,relative-import,no-self-use"
+disabled="--disable=raise-missing-from,line-too-long,wrong-import-order,similarities,invalid-name,too-many-statements,too-many-arguments,too-many-locals,too-few-public-methods,relative-import,no-self-use"
```

Comment:
I'm not saying that we should not add these three parameters but I'm wondering on the reasons to do so.
wrong-import-order seems to be needed
line-too-long, I didn't check the threshold, maybe we can set it higher
raise-missing-from, guess it is because of sqlalchemy but is it still needed if sqlalchemy is ignored?

Reply:
The max line length can be higher if we want.
raise-missing-from was related to the "try.. except" exceptions. The check fails if there is no alias for the object after "except" and when the exception is raised there is something like "from [alias]" but in this case I didn't have any object and I'm not sure how to add it. It could look a bit redundant.

Reply:
I maybe should have added comments in few places, but it's easier here. Have you tried that: https://docs.python.org/3/tutorial/errors.html#raising-exceptions and just use `raise` instead of `raise XxxxxxError...` in your except statement

Reply:
Good idea. I've just enabled the raise-missing-from check and tried that and it worked.